Factory/Config/Filter/BackgroundFactory.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/*
+ * This file is part of the `liip/LiipImagineBundle` project.
+ *
+ * (c) https://github.com/liip/LiipImagineBundle/graphs/contributors
+ *
+ * For the full copyright and license information, please view the LICENSE.md
+ * file that was distributed with this source code.
+ */
+
+namespace Liip\ImagineBundle\Factory\Config\Filter;
+
+use Liip\ImagineBundle\Config\Filter\Type\Background;
+use Liip\ImagineBundle\Config\FilterInterface;
+use Liip\ImagineBundle\Factory\Config\FilterFactoryInterface;
+
+final class BackgroundFactory implements FilterFactoryInterface
+{
+    const NAME = 'background';
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getName(): string
+    {
+        return self::NAME;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function create(array $options): FilterInterface
+    {
+        $color = isset($options['color']) ? $options['color'] : null;
```

Comment:
the most condensed for these checks would be `$color = $options['color'] ?? null;`

Reply:
is it a typo ?
should it be like this ?
`$color = $options['color'] ?: null;`

Reply:
no. that will generate a warning when the index 'color' is not set. ?? is the [null coalesce operator](http://php.net/manual/en/language.operators.comparison.php#language.operators.comparison.coalesce) introduced in PHP 7 to shorten exactly the construct you used above.